src/dotnet/APIView/APIView/Analysis/Analyzer.cs

```diff
@@ -9,7 +9,7 @@
 
 namespace APIView.Analysis
 {
-    internal class Analyzer : SymbolVisitor
+    public class Analyzer : SymbolVisitor

```

Comment:
Is this class use for the .NET analyzers? or is there other functionality to it

Reply:
Yes, it is used for getting the Diagnostics information